src/main/java/edu/harvard/iq/dataverse/api/Metadata.java

```diff
@@ -59,7 +65,35 @@ public Response exportAll() {
     public Response reExportAll() {
         datasetService.reExportAllAsync();
         return this.accepted();
-    } 
+    }
+
+    @GET
+    @Path("reExportDataset")
+    public Response indexDatasetByPersistentId(@QueryParam("persistentId") String persistentId) {
+        if (persistentId == null) {
+            return error(Response.Status.BAD_REQUEST, "No persistent id given.");
+        }
+        Dataset dataset = null;
+        try {
+            dataset = datasetService.findByGlobalId(persistentId);
```

Comment:
Hmm, shouldn't we support both PID and ID (database ID)? We have patterns for this in Datasets.java.